packages/repo-tools/src/commands/util.ts

```diff
@@ -24,14 +24,33 @@ import pLimit from 'p-limit';
 // just per-runner.
 const limiter = pLimit(os.cpus().length);
 
+/**
+ * Quotes an argument if it contains spaces or other characters that need escaping
+ */
+function quoteArgument(arg: string): string {
+  if (!arg) return arg;
+
+  // If the argument contains spaces, wrap it in double quotes
+  if (arg.includes(' ')) {
```

Comment:
Silly but, do we need to check if it already has quotes around it so we don't double them? I'm honestly not sure